src/sentry/utils/eventuser.py

```diff
@@ -125,6 +125,21 @@ def from_snuba(result: Mapping[str, Any]) -> EventUser:
             user_ident=result.get("user_id"),
         )
 
+    @classmethod
+    def for_tags(cls, project_id: int, values):
+        """
+        Finds matching EventUser objects from a list of tag values.
+
+        Return a dictionary of {tag_value: event_user}.
+        """
+        projects = Project.objects.filter(id=project_id)
+        result = {}
+        for value in values:
```

Comment:
How many values are we likely to see here? I'm not totally sure how this is going to be used.

Reply:
We might need to figure out how to make this run in batch so that we don't hit an N+1 during exports.